best-practices/uuid.md

```diff
@@ -0,0 +1,56 @@
+---
+title: UUID Best Practices
+summary: Learn best practice and strategy for using UUID's with TiDB.
+---
+
+# UUID Best Practices
+
+## The usecase for UUID
+
+With TiDB and MySQL UUID's are often used as primary key instead of a integer primary key.
+
+The benefits of UUID's are:
+- UUID's are designed to be globally unique they can be generated on multiple systems without risking conflicts.
+- UUID's are supported by most programming languages and database systems
```

Comment:
I'm not sure I'd say _often_ because it might sound contradictory to other advice. But I'd be okay with saying might be used, or something less common.

```suggestion

There are several benefits to using a `UUID` as a primary key, instead of an `AUTO_INCREMENT` integer value:

- They are designed to be globally unique they can be generated on multiple systems without risking conflicts. In some cases, this means that the number of network trips to TiDB can be reduced, leading to improved performance.
- They are supported by most programming languages and database systems.
- When used as part of URLs, a UUID is not vulnerable to enumeration attacks. i.e. with an `auto_increment` number, it is possible to guess the other invoice ids or user ids.
```

Reply:
Thanks!